code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -77,6 +85,21 @@
 	turf_type = /turf/simulated/floor/plasteel
 	mineralType = "metal"
 
+/obj/item/stack/tile/plasteel/cyborg
+	name = "floor tiles"
+	gender = PLURAL
+	singular_name = "floor tile"
+	desc = "Those could work as a pretty decent throwing weapon."
+	icon_state = "tile"
+	force = 6
+	materials = list(MAT_METAL=500)

```

Comment:
@Fox-McCloud Not emptying the list was a slip-up, but I'm confused. Do I redefine plasteel tiles or not?